### PR TITLE
Simplify build by removing a separate wiremock rule for document mana…

### PR DESCRIPTION
**Pull request by andrewfolga**
…gement.

### JIRA link (if applicable) ###
No JIRA just ad hoc fix.

### Change description ###
Simplify build by removing a separate wiremock rule for document management.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
         <module name="LineLength">
-            <property name="max" value="120"/>
+            <property name="max" value="160"/>
             <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
```
Comment (adr1ancho) [inline config/checkstyle/checkstyle.xml]: Code is difficult to review here once line length exceed github code window size

Comment (andrewfolga) [reply config/checkstyle/checkstyle.xml]: Hmm you can insert this in console so you have wider diffs in github:
`
javascript: document.body.classList.toggle('full-width');
`

Comment (andrewfolga) [reply config/checkstyle/checkstyle.xml]: You can actually make a bookmark in your browser and click it to make the diffs the width of the browser window.

File: src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java [hunk 2]
```diff
@@ -136,7 +128,7 @@
         "    \"AddressLine2\": \"Address Line 12\"\n" +
         "  },\n" +
         "  \"D8Document\":{" +
-        "    \"document_url\": \"http://localhost:%s/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0\"" +
+        "    \"document_url\": \"http://localhost:" + wireMockRule.port() + "/documents/05e7cd7e-7041-4d8a-826a-7bb49dfd83d0\"" +
```
Comment (mario-paniccia) [inline src/test/java/uk/gov/hmcts/ccd/endpoint/std/CallbackTest.java]: can we just have a method port()
which has the details on how to get the port? If that changes again in the future we would also have to change the implementation of that method

**Pull request merged**